### PR TITLE
fix websocket logs

### PR DESCRIPTION
**Pull request by syntrust**

### CODE REVIEW

File: cluster/slave/api.go [hunk 1]
```diff
@@ -132,7 +132,7 @@ func (api *PublicFilterAPI) NewHeads(ctx context.Context, fullShardId hexutil.Ui
 }
 
 // Logs creates a subscription that fires for all new log that match the given filter criteria.
-func (api *PublicFilterAPI) Logs(ctx context.Context, crit rpc.FilterQuery, fullShardId hexutil.Uint) (*rpc.Subscription, error) {
```
Comment (pingke) [inline cluster/slave/api.go]: is this api same as py after change ?

File: cluster/slave/filters/filter_system.go [hunk 2]
```diff
@@ -179,20 +179,11 @@ func (es *EventSystem) subscribe(sub *subscription) *Subscription {
...
-		return nil, err
-	}
-
-	if crit.FromBlock == nil || crit.FromBlock.Int64() == qrpc.LatestBlockNumber.Int64() {
-		crit.FromBlock = big.NewInt(int64(header.Number))
-	}
-	if crit.ToBlock == nil || crit.ToBlock.Int64() == qrpc.LatestBlockNumber.Int64() {
-		crit.FromBlock = big.NewInt(int64(header.Number))
+	if crit.ToBlock == nil {
+		crit.ToBlock = new(big.Int).SetInt64(qrpc.LatestBlockNumber.Int64())
```
Comment (pingke) [inline cluster/slave/filters/filter_system.go]: please check this value -- qrpc.LatestBlockNumber.Int64(), is it -1?

File: cluster/slave/api.go (hunk 1 above)
Comment (syntrust) [reply cluster/slave/api.go]: this fix aim to be same with py

File: cluster/slave/filters/filter_system.go (hunk 2 above)
Comment (syntrust) [reply cluster/slave/filters/filter_system.go]: yes. it dynamicly points to the current block

**Pull request merged**